mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -114,6 +115,12 @@ func parseKubectlShortVersion(version string) ([3]int, error) {
 
 // IsMesheryRunning checks if the meshery server containers are up and running
 func IsMesheryRunning(currPlatform string) (bool, error) {
+	// Checking if Meshery is running with "make run-local" or "make run-fast"
+	resp, _ := http.Get("http://localhost:9081/api/server/version")
+
+	if resp.StatusCode == 200 {
```

Comment:
```suggestion
	if resp != nil && resp.StatusCode == 200 {
```
This is the nil reference thing.